instrumentation/net/http/otelhttp/common.go

```diff
@@ -31,10 +31,16 @@ const (
 
 // Server HTTP metrics.
 const (
-	RequestCount          = "http.server.request_count"           // Incoming request count total
-	RequestContentLength  = "http.server.request_content_length"  // Incoming request bytes total
-	ResponseContentLength = "http.server.response_content_length" // Incoming response bytes total
-	ServerLatency         = "http.server.duration"                // Incoming end to end duration, milliseconds
+	ServerRequestSize  = "http.server.request.size"  // Incoming request bytes total
+	ServerResponseSize = "http.server.response.size" // Incoming response bytes total
+	ServerDuration     = "http.server.duration"      // Incoming end to end duration, milliseconds
+)
+
+// Client HTTP metrics.
+const (
+	ClientRequestSize  = "http.client.request.size"  // Outgoing request bytes total
+	ClientResponseSize = "http.client.response.size" // Outgoing response bytes total
+	ClientDuration     = "http.client.duration"      // Outgoing end to end duration, milliseconds
```

Comment:
Can we unexport these constants and add a changelog entry in "Removed" section?

```
- Remove `RequestCount`, `RequestContentLength`, `ResponseContentLength`, `ServerLatency` constants from `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#4707)
```

Reply:
233049a5211e6aebcbdd90d6fd4fe4d50e114f49

Reply:
Server* constans have to be unexported as well.

Reply:
My bad, I misunderstood. Will take care of that when I get back to the computer.